lib/ditcher.js

```diff
@@ -590,6 +590,9 @@ Ditcher.prototype.doImport = function (params, cb) {
     var collectionsToImportArray = Object.keys(collectionsToImport);
 
     collectionsToImportArray.forEach(function(colKey){
+      if (collectionsToImport[colKey].length === 0) {
+        return cb(new Error("collection " + colKey + " is empty"));
```

Comment:
We are sending an error and aborting execution currently for duplicate key error https://github.com/rachael-oregan/fh-db/blob/1229d57d01f28d7e8c7801cab88d72ae85b6537b/lib/ditcher.js#L610
maybe we can add this check into the importers which will mirror current functionality?